src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -8,38 +8,31 @@ import { Loading } from '../../Loading/Loading';
  * @param { import('../../../services/pure-di').AppServices } props.dependencies
  */
 const ReportsBox = ({ dateFrom, dateTo, emailFilter, today, visits, loading }) => {
+  const withEmail = 'with_email' === emailFilter;
+  const totalVisits = visits ?? 0;
+  const descriptionKey = withEmail
+    ? 'reports_box.visits_description_with_email'
+    : 'reports_box.visits_description_without_emails';
+  const titleKey = withEmail
+    ? 'reports_box.visits_with_email'
+    : 'reports_box.visits_without_emails';
```

Comment:
I suppose that these changes are part of the behavior change. It would be nice to have them in a clean commit to allow the reviewers to see what was changed at a glance.